devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
I have seen with healthcheck, that I thought there was suppose to be a ping() method that it calls if something is amiss.   My understanding is if things are ok, before the timeout something needs to update an event variable, otherwise health check won't know.
So it seems you need to ensure on every timer running, some variable (even a last timer run) needs to be updated.

It is hard to know for sure, as they don't really claim it to be supported for "public use"